app/frontend/components/pages/login/aboutOverlay.js

```diff
@@ -20,152 +21,129 @@ class AboutOverlayClass extends Component {
+                'https://shop.trezor.io/product/trezor-model-t?offer_id=15&aff_id=1071&source=Disclaimer',
+              target: '_blank',
+            },
+            'Trezor model T'
           ),
+          '. ',
           h(
-            'div',
+            'b',
             undefined,
-            h(
-              'label',
-              {class: 'centered-row action'},
-              h('input', {
-                type: 'checkbox',
-                checked: dontShowAgainCheckbox,
-                onChange: this.checkboxClick,
-                class: 'understand-checkbox',
-              }),
-              'I understand the risk and do not wish to be shown this screen again'
-            )
+            `Trezor is a hardware wallet that will allow you to store your
+                mnemonic and private keys safely and interact with AdaLite in the safest possible
+                manner, without giving away to AdaLite's interface your mnemonic nor root private key.`
           ),
-          h(
-            'span',
-            {class: 'centered-row'},
-            h(
-              'button',
-              {
-                onClick: this.toggleAboutOverlay,
-                class: 'rounded-button',
-                autofocus: true,
-                onKeyDown: (e) => {
-                  e.key === 'Enter' && e.target.click()
-                  e.key === 'Tab' && e.preventDefault()
-                },
-              },
-              'Close'
-            )
-          )
+          ` Your private key will never leave the Trezor. Therefore an attacker can’t steal
+              them and gain control over your wallet.`
+        ),
+        h(
+          'p',
+          undefined,
+          'To protect yourself from phishers, bookmark the official AdaLite address and always check the URL ',
+          h('code', undefined, h('u', undefined, 'https://adalite.io/')),
+          '.'
+        ),
+        h(
+          'p',
+          undefined,
+          'AdaLite was not created by Cardano Foundation, Emurgo, or IOHK. This project was created with passion by ',
+          h('a', {href: 'https://vacuumlabs.com/', target: '_blank'}, 'Vacuum Labs'),
+          '. We appreciate any feedback, donation or contribution to the ',
+          h('a', {href: 'https://github.com/vacuumlabs/adalite/', target: '_blank'}, 'codebase'),
+          '.'
+        )
+      ),
+      h('p', undefined, 'Feedback and contributions are very welcome.'),
+      h(
+        'div',
+        undefined,
+        h(
+          'label',
+          {class: 'centered-row action'},
+          h('input', {
+            type: 'checkbox',
+            checked: dontShowAgainCheckbox,
+            class: 'understand-checkbox',
+          }),
+          'I understand the risk and do not wish to be shown this screen again'
+        )
+      ),
+      h(
+        'span',
+        {class: 'centered-row'},
+        h(
+          'button',
+          {
+            onClick: toggleAboutOverlay,
```

Comment:
this should be this.toggleAboutOverlay, otherwise the checkbox won't be saved

Reply:
This, and I also accidentally deleted the line, that changed the checkbox state...